src/main/resources/cassandratablecolumn.properties

```diff
@@ -0,0 +1,14 @@
+id=id
```

Comment:
Add few lines of comment mentioning what is this file and how to use it.. 
It should mention purpose of the key and value.